l10n_eu_nace/models/res_partner_nace.py

```diff
@@ -45,9 +45,9 @@ def name_get(self):
 
     @api.model
     def name_search(self, name, args=None, operator='ilike', limit=100):
-        return self.search(
-            ['|', ('name', operator, name), ('code', 'ilike', name)]
-        ).name_get()
+        args = list(args or [])
+        args += ['|', ('name', operator, name), ('code', 'ilike', name)]
```

Comment:
Oops, I forgot this nasty one:

```suggestion
        args = ['|', ('name', operator, name), ('code', 'ilike', name)] + args
```